system-container/root/exports/manifest.json

```diff
@@ -0,0 +1,11 @@
+{
+    "version": "1.0",
+    "defaultValues": {
+        "OPTS": "",
+        "VAR_LIB_OPENSHIFT_INSTALLER" : "/var/lib/openshift-installer",
+        "VAR_LOG_OPENSHIFT_LOG": "/var/log/ansible.log",
+        "PLAYBOOK_FILE": "/usr/share/ansible/openshift-ansible/playbooks/byo/config.yml",
```

Comment:
Aha, thanks @codificat, I haven't realized that before.

I'd still ask, since that file is a template, perhaps CWD could be part of the template?

I like to find simplicity, if we end up concluding that symlinks make it simpler, I'd favor it. But I'd consider the alternatives first -- update all image Dockerfiles to use the same paths and no need for symlinks; template the CWD in the config file, ...

Being able to provide a relative path makes the command-lines to interact with the images shorter, and relative to the repo root.